pyani_plus/workflows/snakemake_anib.smk

```diff
@@ -82,12 +86,12 @@ rule blastn:
         "{outdir}/{genomeA}_vs_{genomeB}.tsv",
     shell:
         """
-        chronic {params.blastn} -query {wildcards.outdir}/{wildcards.genomeA}-fragments.fna \
+        {params.blastn} -query {wildcards.outdir}/{wildcards.genomeA}-fragments.fna \
             -db {wildcards.outdir}/{wildcards.genomeB} -out {output} -task blastn \
             -outfmt '6 qseqid sseqid length mismatch pident nident qlen slen \
                      qstart qend sstart send positive ppos gaps' \
-            -xdrop_gap_final 150 -dust no -evalue 1e-15 -max_target_seqs 1 &&
-        chronic .pyani-plus-private-cli log-anib --database {params.db} \
+            -xdrop_gap_final 150 -dust no -evalue 1e-15 > {output}.log &&
```

Comment:
We could keep -max_target_seqs 1 (if we wanted) and redirect STDERR to /dev/null or something.